wallet-new/server/Cardano/Wallet/API/V1/Swagger/Example.hs

```diff
@@ -14,6 +15,7 @@ import           Pos.Util.BackupPhrase (BackupPhrase)
 import           Pos.Wallet.Web.ClientTypes (CUpdateInfo)
 import           Pos.Wallet.Web.Methods.Misc (WalletStateSnapshot (..))
 
+import           Data.Map.Strict (fromList)
```

Comment:
I'd rather see this imported qualified; `fromList` is defined on so many things, and while the explicit import makes it unambiguous, you have to jump to the import list to figure that out.

Reply:
Very good point :+1: